contracts/test/contract/ethCallAware.spec.ts

```diff
@@ -38,41 +38,56 @@ describe("EthCallAware", async () => {
   const data = "0x2020";
 
   const tests = async (skipEthCallAware: boolean) => {
-    it(`allows transaction to continue (skipped: ${skipEthCallAware})`, async () => {
-      const ethCallAware = await setupEthCallAware();
-      const res = await ethCallAware.functions.testFunction(num, data, skipEthCallAware);
-      const receipt = await res.wait();
+    for (let i = 0; i < 2; i++) {
+      const opts: any = {};
+      if (!skipEthCallAware) {
```

Comment:
I think we can actually get rid of the skipEthCallAware tests altogether. We used to pass that skip parameter down into EthCallAware so we did want to test it, but now skip is just a feature of the Tester contract which we dont need to test.

We should also remove the skip param from the tester contract too?

Reply:
yup